### ISSUE TITLE
[Bug]: Mobs always angry with "aggressiveAnimals = false"

### ISSUE THREAD

**Issue by Soforak** [Mrbysco/AngryMobs#3]
### General Info

- [X] I am running a modpack
- [X] I can reproduce this issue consistently in single-player
- [X] I can reproduce this issue consistently in multi-player
- [X] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I am having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [X] I am using the latest version of the mod

### Forge version

21.1.42

### Minecraft version

1.21.1

### Mod version

7.0.0

### Java version

21

### Issue Description

Hi, I tried every options without success.
Vanilla aimals are alwayss agressive as long as angryAnimals is true, regardles of other parameters.

```
#Common settings
[Common]
	#When true makes all vanilla animals attack you if you hurt them
	angryAnimals = true
	#When true makes all vanilla animals attack you on sight
	aggressiveAnimals = false
	#When true makes the melee goals add the mobs attack damage and knockback onto the specified values
	useAttributes = false

``` 

Also, I can't figure how to use other options such as HurtByTarget. Where to put them?

It would be really great if I could use this mod without problem!
Thanks

**Comment by Mrbysco** [comment]
Any custom configuration requires using crafttweaker or the reipe system

**Comment by Soforak** [comment]
Sorry I didn't express myself correctly
The problem is that mobs don't become agressive when hit but are actively seeking to kill me even with agressiveAnimal to false

Aren't they supposed to only want to kill me if I hurt them like piglins?

**Comment by Mrbysco** [comment]
Did you make a new world after turning the config off? Or spawn a fresh mob? as it won't retroactively change AI goals

**Comment by Soforak** [comment]
Ok, so to be sure, I just created a new intance with only angryMobs and after trying to change settings, spawning new mobs, I can confirm that if angryAnimals = false, they don't attack and with angryAnimals = true, they do, but regardless of hitting them, they always attack even with agressiveAnimals = false

**Comment by Mrbysco** [comment]
I'll look into it

**Comment by Mrbysco** [comment]
I fixed the issue, 7.0.1 will be uploaded shortly and available within the next hour

**Comment by Soforak** [comment]
Tested and can confirm everything works fine now, thanks!
